packages/payment-processor/overrides/jest/index.d.ts

```
@@ -0,0 +1,1779 @@
+    [K in keyof RemoveIndex<T> as RemoveIndex<T>[K] extends Constructor
+      ? K
+      : never]: RemoveIndex<T>[K];
+  };
+  type ConstructorPropertyNames<T> = RemoveIndex<keyof ConstructorProperties<T>>;
+
+  interface DoneCallback {
+    (...args: any[]): any;
+    fail(error?: string | { message: string }): any;
+  }
+
+  type ProvidesCallback = ((cb: DoneCallback) => void | undefined) | (() => PromiseLike<unknown>);
```

Biome

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)